src/makeRobot.js

```diff
@@ -39,6 +39,63 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  return {
+    name: name,
+    version: version,
+    wheels: wheels,
```

Comment:
If var === key, you can use short version
```suggestion
    name,
    version,
    wheels,
```